Loenn/entities/crate_spawner.lua

```diff
@@ -0,0 +1,56 @@
+local drawableRectangle = require("structs.drawable_rectangle")
+local drawableSprite = require("structs.drawable_sprite")
+
+local crateSpawner = {}
+
+crateSpawner.name = "FactoryHelper/ThrowBoxSpawner"
+
+crateSpawner.fieldInformation = {
+    delay = {
+        minimumValue = 0.0
+    },
+    maximum = {
+        fieldType = "integer",
+        minimumValue = 0
+    }
+}
+
+crateSpawner.placements = {
+    {
+        name = "crate_spawner",
+        data = {
+            delay = 1.0,
+            maximum = 0,
+            activationId = "",
+            isMetal = false,
+            isRandom = false,
+            fromTop = true,
+            tutorial = false,
+            startActive = true
+        }
+    }
+}
+
+local normalColor = {1.0, 1.0, 1.0, 0.2}
+local randomColor = {1.0, 0.5, 1.0, 0.2}
+
+function crateSpawner.sprite(room, entity)
```

Comment:
I kinda liked the arrow indicator on the Ahorn plugin to show it'd drop downwards, not a huge deal though

Reply:
i would either need a sprite added to the mod exclusively for indicating that it drops down, or a way to draw arrows

Reply:
I'll just make a note to add them after Lonn arrows happen lol